tk_toolchain/cmd_line_tools/tk_docs_generation/__main__.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
```

Comment:
Ensures we can run `tk-docs-preview` using `python -m tk_toolchain.cmd_line_tools.tk_docs_generation`